File: MS_lifehealthservices/LHSAPI.Domain/Entities/EmployeeJobProfile.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace LHSAPI.Domain.Entities
{
    public class EmployeeJobProfile : BaseEntity
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        public int EmployeeId { get; set; }

        public string JobDesc { get; set; }

        public int? DepartmentId { get; set; }
        public int? ReportingToId { get; set; }

        public DateTime? DateOfJoining { get; set; }

        public int? Source { get; set; }
        public int? WorkingHoursWeekly { get; set; }

        public int? DistanceTravel { get; set; }

        
    }

}

File: MS_rosterfrontend/projects/employee-detail/src/lib/view-models/employee-jobprofile.ts
export interface EmployeeJobProfile {
    id?: number;
    employeeId?: number;
    jobDesc?: string;
    departmentId?: number;
    departmentName?: string;
    locationId?: number;
    locationName?: string;
    reportingToId?: number;
    reportingToName?: string;
    dateOfJoining?: string;
    source?: number;
    sourceName?: string;
    workingHoursWeekly?: number;
    distanceTravel?: number;
    locationType?: number;
    otherLocation?:string;
    locationTypeName?:string;

}